### ISSUE TITLE
Add line break on character limit for Title column on table

### ISSUE THREAD

**Issue by yankeexe** [yankeexe/good-first-issues#15]
When the terminal window is of minimal length, the table gets jumbled up.  Sometimes it's due to the long length of title on GitHub issues. Can it be solved by adding a check for character limit on `issue title` column and adding a line break? 

If you have been using this tool, let me know if you have faced this issue or have any solutions in your mind including the one mentioned above. 

Image for reference: 
![image](https://user-images.githubusercontent.com/13623913/95697374-43ddca80-0c5e-11eb-99d1-266b487da706.png)

**Comment by kra-tosss** [comment]
@yankeexe I would like to work on this , could you assign this to me ?

**Comment by yankeexe** [comment]
Hey @kra-tosss sure. Do you have any suggestions on how much should we set the character limit?

**Comment by akshitagupta20** [comment]
hey @yankeexe about this issue, I think the character limit can be 50 characters then we can add a line break then again the character limit for the second line and at last a line break.
what do you think?

**Comment by yankeexe** [comment]
> hey @yankeexe about this issue, I think the character limit can be 50 characters then we can add a line break then again the character limit for the second line and at last a line break.
> what do you think?

yeah that sounds great. Do you want to work on it? @akshitagupta20

**Comment by bhatt01tapasvi** [comment]
Just one doubt, what if a 50 character limit is reached when we are in the middle of a string? What I was thinking is that we can add the line break there with a hyphen showing that the word is continued in the next line. Can I work on this?

**Comment by yankeexe** [comment]
> Just one doubt, what if a 50 character limit is reached when we are in the middle of a string? What I was thinking is that we can add the line break there with a hyphen showing that the word is continued in the next line. Can I work on this? @yankeexe 

Yeah that works, but I think you can add a line break before the word if a word crosses the limit on the middle. 

You can work on this @bhatt01tapasvi  👌

**Comment by bhatt01tapasvi** [comment]
Sure. Thanks.

**Comment by bhatt01tapasvi** [comment]
I am done with the code. I am confused with which file to make changes to. Can you please tell me which file is it? @yankeexe

**Comment by yankeexe** [comment]
Hey @bhatt01tapasvi  you can look into these lines 
https://github.com/yankeexe/good-first-issues/blob/dab3c3adbc3a4170af17eaa36fa788bf02fe9bdf/good_first_issues/graphql/commands.py#L92-L99

Issues contains the List of issues with their URL as a tuple: 
`issues = [(issue title, url), (issue title, url)]`

You can extract the Issue title and work on them.

**Comment by behumoury** [comment]
hey, I have just started coding and would love to contribute here. But I am a bit confused about how to and where to start. Can someone brief me, pls

**Comment by ammarfitwalla** [comment]
Hi , I see it's still in open state, can I work on this if it is not resolved yet?

**Comment by anubhav1206** [comment]
Can i work on this issue?

**Comment by yankeexe** [comment]
@anubhav1206 sure! Go for it ⚡

**Comment by SaboorNisha** [comment]
You can Set a Character Limit and Add Line Breaks to solve this issue .
you can use function when working on CLI Application 
like ,
SELECT 
    CASE 
        WHEN LENGTH(title) > 50 THEN 
            CONCAT(SUBSTRING(title, 1, 50), '\n', SUBSTRING(title, 51)) 
        ELSE title 
    END AS formatted_title
FROM issues_table;

**Comment by ANKITHASBHAT** [comment]
Please assign the issue to me @yankeexe :)

**Comment by Rocklinks** [comment]
@yankeexe  if the issue still persists can i give it a try

**Comment by P3st83** [comment]
Hi can i give it  a try?

**Comment by sakshi1110g** [comment]
@yankeexe I would like to work on this , could you please assign this to me ?

**Comment by Rameshkumar31595** [comment]
Hi! 👋 I noticed this issue is still open. Is it still relevant?
If yes, I’d love to work on this as part of Hacktoberfest.

**Comment by myselfchandrakantmandal-creator** [comment]
Hey I want to work on it .

**Comment by jinethonazam-droid** [comment]
Hi! I'm a student and I'm new to this project. I would like to work on this issue. Could you please assign it to me?

**Comment by Abd7allah** [comment]
Hi, I’d like to work on this issue. Is it still available?

**Comment by keshavsharma0614-blip** [comment]
Hi! I’ve submitted the fix for this issue. Since this is my first contribution, the workflow is currently awaiting maintainer approval. Please let me know if any changes are needed—happy to update the PR.

**Comment by premsankar5119-spec** [comment]
Hi, I would like to work on this issue. I have reviewed the problem and would like to contribute a fix. Could you please assign this issue to me?

**Comment by prabhavofficial07** [comment]
Hey I would Like to work on this Issue Can you Please Assign this to me
